### ISSUE TITLE
Invoker Ice Wall doesn't take slow resistance into account

### ISSUE THREAD

**Issue by nelsonchacko98** [ValveSoftware/Dota2-Gameplay#10708]
### Ability name

Ice Wall

### Description

Invoker Icewall is not taking slow resistance into account and is always slowing the hero for the same amount.

**Movement speed before rocket barrage**
![Uploading Screenshot (23).png…]()

**Movement speed after using rocket barrage; There is no change**
![Screenshot (24)](https://github.com/ValveSoftware/Dota2-Gameplay/assets/23289511/750ac422-0e34-4064-84f7-aad8a115dfc4)

### Example Match ID (and possibly Timestamp)

None; Tested it in lobby

**Comment by LokeLok** [comment]
This is not correct, Ice Wall does take slow resistance into account. This is just a visual UI bug, go see and upvote #139. If you wanna test this yourself, get Gyrocopter some movement speed(500+ ms) items and use Rocket Barrage while you run up and down the Ice Wall. You will see him speeding up when you use Rocket Barrage(yes, this is without the 10 talent(right)).
